components/detail/SwapDetailSummary/SwapDetailMobileValue.tsx

```diff
@@ -26,6 +29,17 @@ function SwapDetailMobileValue(props: SwapDetailItem) {
               <div className="flex items-center">
                 <StepsIcon className="text-neutral-400 mr-5" />
                 <span className="text-12 text-neutral-400">{`${successStep}/${steps.length}`}</span>
+                &nbsp;&nbsp;
+                {swapper && (
+                  <Tooltip label={swapper?.swapperTitle}>
+                    <img
```

Comment:
NIT: use `next/image` instead of `<img>`.

Reply:
fixed